org.eclipse.lsp4e/src/org/eclipse/lsp4e/LanguageServers.java

```diff
@@ -268,6 +269,42 @@ public long getStartVersion() {
 			return this.startVersion;
 		}
 
+		public SingleServerDocumentExecutor toExecutor(final @NonNull LanguageServerWrapper serverWrapper) {
+			return new SingleServerDocumentExecutor(this.document, serverWrapper);
+		}
+	}
+
+	/**
+	 * Executor for a single server - for follow-up requests that should go to the server that returned an earlier response
+	 *
+	 */
+	public static class SingleServerDocumentExecutor extends LanguageServerDocumentExecutor {
```

Comment:
I think that to be consistent with the names of the other classes, it should be named `SingleLanguageServerDocumentExecutor`, in the name of the other classes we use `LanguageServer` and not just `Server`

Reply:
Done. The names are becoming a bit cumbersome but I guess that's what `var` was invented for ;-)